compose/local/env

```diff
@@ -0,0 +1,70 @@
+APP_NAME=Restarters.net
+APP_ENV=local
+APP_KEY=base64:KX7NwPdsechAghSsyjbzW+i/lVV81iV7TjY8ZjyNric=
```

Comment:
Wondering if keys should be generated as part of the Docker setup?
with php artisan key:generate.

I think they should be unique per instance of the app (although not 100% sure)

Reply:
Yeah, I can add a step for that...